### DISCUSSION TITLE
0.16.4/5 does not build on rPI 5

### DISCUSSION THREAD

**Discussion by RotateAt60MPH** [OvenMediaLabs/OvenMediaEngine#1614] [Help]
**Describe the bug**
OvenMediaEngine-0.16.4.tar.gz does not build (on rPI 5)

**To Reproduce**
1. Download and flash Raspberry PI OS 64 bit to rPI 5.
2. Download OvenMediaEngine-0.16.4.tar.gz from github
3. run 'OvenMediaEngine-0.16.4/misc/prerequisites.sh'
4. fails.  Prereq packages utilities do not exist, like autoconf.
5. Followed troubleshooting guide. 
    sudo apt install -y build-essential nasm autoconf libtool zlib1g-dev tclsh cmake curl
6. Was able to re-run the prerequisites.sh script successfully with no intervention
7. run 'make release'
8. Fails with
projects/base/ovlibrary/uuid.h:14:10: fatal error: uuid/uuid.h: No such file or directory

Same build problem experienced with OvenMediaEngine 0.16.5.
0.16.4 built and runs fine on rPI 4.

**Expected behavior**
Build from source should work as expected

**Logs**
```
admin@raspberrypi:~/OvenMediaEngine-0.16.4/src $ make release
####################################################
#                                                  #
#           AirenSoft Make System v0.63            #
#                                                  #
####################################################
   - Operating system        : linux
   - Core count              : 4
   - GCC version             : 12.2.0
   - Build option            : RELEASE
   - Paths for linker        : /opt/ovenmediaengine/lib:/opt/ovenmediaengine/lib64
   - Paths for pkg-config    : /opt/ovenmediaengine/lib/pkgconfig:/opt/ovenmediaengine/lib64/pkgconfig
   - Total Projects count    : 62

[  1/504|  0%] Preparing directories...
    Preparing directory bin/RELEASE...
    Preparing directory intermediates/RELEASE...
    Preparing directory intermediates/RELEASE/objs...
    Preparing directory intermediates/RELEASE/static...
[  2/504|  0%] [static] libapi_server.a: C++ projects/api_server/api_server.cpp => intermediates/RELEASE/objs/api_server/api_server.o
In file included from projects/config/items/server.h:14,
                 from projects/config/items/items.h:25,
                 from projects/config/config_manager.h:11,
                 from projects/config/config.h:12,
                 from projects/base/info/host.h:11,
                 from projects/api_server/api_server.h:11,
                 from projects/api_server/api_server.cpp:9:
projects/base/ovlibrary/uuid.h:14:10: fatal error: uuid/uuid.h: No such file or directory
   14 | #include <uuid/uuid.h>
```

**Server (please complete the following information):**
 - OS: Linux raspberrypi 6.6.20+rpt-rpi-2712 #1 SMP PREEMPT Debian 1:6.6.20-1+rpt1 (2024-03-07) aarch64 GNU/Linux
 - OvenMediaEngine Version: v0.16.4
 - Branch: n/a

**Player (please complete the following information):**
- N/A

**Additional context**
- None

**Comment by RotateAt60MPH** [comment]
It appears you need to add the uuid-dev package in order for the build to progress.  Will update the issue as I investigate the build process further.

**Comment by getroot** [comment]
Since we don't officially support rPI I'm moving this to discussion.
